src/core/Memory.ts

```diff
@@ -1,9 +1,9 @@
-let values_in_browser: { date: Date, value: number }[] = [];
+export const storageKey = 'values';
 
 export function saveValues(values: { date: Date, value: number }[]): void {
-    values_in_browser = values;
+    localStorage.setItem(storageKey, JSON.stringify(values))
 }
 
 export function getValues(): { date: Date, value: number }[] {
-    return values_in_browser;
+    return JSON.parse(localStorage.getItem(storageKey) || '[]')
```

Comment:
Just a tip.
instead of '||' you can use '??'.
?? checks left side for undefined only.

Reply:
Thanks :) i didn't know